video/out/gpu/hwdec.h

```diff
@@ -25,7 +25,7 @@ void ra_hwdec_ctx_init(struct ra_hwdec_ctx *ctx, struct mp_hwdec_devices *devs,
 void ra_hwdec_ctx_uninit(struct ra_hwdec_ctx *ctx);
 
 void ra_hwdec_ctx_load_fmt(struct ra_hwdec_ctx *ctx, struct mp_hwdec_devices *devs,
-                           int imgfmt);
+                           int imgfmt, bool probing);
```

Comment:
If we move the struct to here, we could use it in this signature and avoid having to enumerate the probing flag in every function call